src/referencePackages/Directory.Build.props

```diff
@@ -0,0 +1,92 @@
+<Project>
+
+  <!--
+    The project name uses {AssemblyName}.{Version}.csproj format to be unqiue, because we build
+    multiple versions of the same package IDs. This means the AssemblyName will include the version
+    in the name, unlike the real packages. To fix this, use the parent parent dir name as the
+    assembly name, as it has the correct versionless name and capitalization.
+
+    If this doesn't work in a specific case, manually define AssemblyName in the project, or in a
+    Directory.Build.props for that whole package ID.
+  -->
+  <PropertyGroup>
+    <ProjectParentDir>$([System.IO.Directory]::GetParent('$(MSBuildProjectDirectory)'))</ProjectParentDir>
+    <AssemblyName Condition="'$(AssemblyName)' == ''">$([MSBuild]::MakeRelative('$(MSBuildThisFileDirectory)src', '$(ProjectParentDir)'))</AssemblyName>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
+
+  <PropertyGroup>
+    <!-- Common properties for all Reference Packages -->
+    <DebugType>None</DebugType>
+    <DebugSymbols>false</DebugSymbols>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <AssemblyOriginatorKeyFile>$(RepoRoot)src/referencePackages/MSFT.snk</AssemblyOriginatorKeyFile>
```

Comment:
This file doesn't exist?  Move below next line and use `$(KeyFileDir)`?  Or is this a catch in the case that the project doesn't define `AssemblyOriginatorKeyFile`?  If the latter, maybe a comment stating that?

Reply:
This is just moved from here, not sure why it exists--the file doesn't exist in master either:

https://github.com/dotnet/source-build-reference-packages/blob/aa0b7435c220ae8ef247e775f32eac3dbeb7162c/src/referencePackages/src/dir.props#L11

I'll try removing it.